CMake/BuildPeleCExe.cmake

```diff
@@ -69,6 +108,7 @@ function(build_pelec_exe pelec_exe_name pelec_lib_name)
        ${SRC_DIR}/React.cpp
        ${SRC_DIR}/Riemann.H
        ${SRC_DIR}/Setup.cpp
+       ${SRC_DIR}/Soot.cpp
```

Comment:
Should we put this include into a `PELEC_USE_SOOT` if condition? Or would that create a mess?

Reply:
I was under the impression that CMake knows not to worry about it unless it is needed. I could be wrong. But I haven't had an issue with it being there when doing CMake stuff without soot.